### PR TITLE
Ferhat-cake-page

### PR DESCRIPTION
**Pull request by ferhatu**

### CODE REVIEW

**Comment by ferhatu** [comment]
thanks

File: index.html [hunk 1]
```diff
@@ -6,11 +6,94 @@
...
+    <header class= "header">
+            <div>
+                <img class="logo" src="images/cake-logo.png" alt="Logo Image" >
+            </div>
+            <div>
+                <i class="fa fa-envelope"></i>
+            </div>
+    </header>
+    <nav>  
+        <div class="topnav">
```
Comment (AlfioEmanueleFresta) [inline index.html]: `<ul>`s and `<li>`s (short for ***u**nordered **l**ist*, and ***l**ist **i**tem*) are great semantic tags for most lists, including list of links in navigation bars, e.g.:

```html
<nav>
  <ul class="topnav">
    <li><a href="#">Home</a></li>
    <li><a href="#">Cakes</a></li>
    ...
  </ul>
</nav>
```

File: index.html [hunk 2]
```diff
@@ -6,11 +6,94 @@
...
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <main>
```
Comment (AlfioEmanueleFresta) [inline index.html]: The `<main>` section should not include the `<header>`, e.g.:

```html
<html>
  <body>
    <header>...</header>
    <main>
      <section>...</section>
      <section>...</section>
    </main>
    <footer>...</footer> 
  </body>
</html>
```

File: style.css [hunk 3]
```diff
@@ -1 +1,179 @@
...
+}
+.row{
+    display: flex;
+    width: 100%;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    
+}
+.rightcolumn{
+    float:right;
```
Comment (AlfioEmanueleFresta) [inline style.css]: You don't need `float`s

File: style.css [hunk 4]
```diff
@@ -1 +1,179 @@
...
+.leftcolumn{
+    float:left;
+    width:28%;
+    height: auto;
+}
+.bottom-box{
+    display:flex;
+    justify-content: space-evenly;
+}
+.box1, .box2, .box3, .box4{
```
Comment (AlfioEmanueleFresta) [inline style.css]: Remember the difference between CSS classes and IDs: the great thing about classes is that they can be applied to multiple elements. If you still *need* to distinguish between boxes, you can also give them a unique ID.

For example, you could have:

```html
<div class="box" id="box1">...</div>
<div class="box" id="box2">...</div>
<div class="box" id="box3">...</div>
...
```

Notice how all elements share a common class (`.box`) but have unique identifiers (`#box1`, `#box2`, `#box3`). As you want to style all elements, you can apply your rules directly to the common `.box` class, e.g.:

```css
.box {
  ...
}
```

**Comment by AlfioEmanueleFresta** [comment]
@ferhatu the page is already looking good, well done! I've added some feedback above - keep working on this! 👍

**Pull request closed without merging**